api/problem_details.py

```diff
@@ -184,6 +184,13 @@
     detail=_("The OAuth callback must contain a code and a state parameter with the OAuth provider name."),
 )
 
+UNKNOWN_OAUTH_PROVIDER = pd(
+    "http://librarysimplified.org/terms/problem/unknown-oauth-provider",
+    status_code=400,
+    title=_("Unknown OAuth provider."),
+    detail=_("The specified OAuth provider name isn't one of the known providers."),
```

Comment:
When you use this would it be useful to .detail() it with a list of the providers that are known?